Controller/UpwardControllerFactory.php

```diff
@@ -40,10 +48,10 @@ public function __construct(ObjectManagerInterface $objectManager)
      */
     public function create(RequestInterface $request): UpwardController
     {
-        $upwardConfig = getenv(static::VAR_UPWARD_CONFIG);
+        $upwardConfig = $this->deploymentConfig->get(self::UPWARD_CONFIG_PATH);
 
         if (empty($upwardConfig)) {
-            throw new \RuntimeException('Environment variable ' . static::VAR_UPWARD_CONFIG . 'not set.');
+            throw new \RuntimeException('Environment variable ' . static::VAR_UPWARD_CONFIG . ' not set.');
```

Comment:
Need to fix the constant name, and should probably update the message. (Thanks for fixing the missing space, I was annoyed at myself for leaving it off)